docs/glossary.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Glossary"
+slug: "/contributor/catalyst/glossary"
+---
+
+### Catalyst
+
+[Catalyst](https://github.com/decentraland/catalyst) is the name given to the **Decentraland Service Bundle**, each Catalyst [node](https://decentraland.github.io/catalyst-monitor) has a set of services that work as the backbone for the platform and exposes a public [API](https://decentraland.github.io/catalyst-api-specs/). 
```

Comment:
The only thing that is not super exact here is the fact that all comms, both v2 and v3 are outisde of catalyst

Reply:
change it to expose